apps/site/assets/ts/hooks/usePredictionsChannel.ts

```diff
@@ -91,7 +91,7 @@ const usePredictionsChannel = (
     PredictionWithTimestamp[] | null,
     ChannelPredictionResponse
   > = (_oldGroupedPredictions, detail) => {
-    if (detail.error) {
+    if (detail.error || !detail.predictions) {
```

Comment:
Disregard, I ended up separating this into a hotfix. But noticed it while working on this because the router's very unforgiving of errors...